.clang-format

```diff
@@ -4,14 +4,14 @@ IncludeCategories:
   - Regex:           '^"'
     Priority:        1
 
-AlignAfterOpenBracket: DontAlign
+AlignAfterOpenBracket: AlwaysBreak
 AlignConsecutiveAssignments: true
 AlignConsecutiveDeclarations: true
 AlignEscapedNewlinesLeft: false
 AlignOperands:   true
 AlignTrailingComments: true
 AlignConsecutiveMacros: true
-AllowAllParametersOfDeclarationOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: true
```

Comment:
I saw the same changes in fyi-pack, is there a new format that has to be applied to all the repos?

Reply:
Yes... :) better to have same settings for all projects...